core/tauri/src/updater/core.rs

```diff
@@ -826,173 +832,69 @@ fn copy_files_and_run<R: Read + Seek>(
-              // Rename the MSI to the match file name the Skip UAC task is expecting it to be
-              let temp_msi = tmp_dir.with_file_name(bin_name).with_extension("msi");
-              Move::from_source(&found_path)
-                .to_dest(&temp_msi)
-                .expect("Unable to move update MSI");
-              let exit_status = Command::new("schtasks")
-                .arg("/RUN")
-                .arg("/TN")
-                .arg(update_task_name)
-                .status()
-                .expect("failed to start updater task");
-
-              if exit_status.success() {
-                // Successfully launched task that skips the UAC prompt
-                exit(0);
-              }
-            }
-            // Failed to run update task. Following UAC Path
-          }
-        }
-      }
-
-      // we need to wrap the current exe path in quotes for Start-Process
-      let mut current_executable = std::ffi::OsString::new();
-      current_executable.push("\"");
-      current_executable.push(dunce::simplified(&current_exe()?));
-      current_executable.push("\"");
-
-      let mut msi_path = std::ffi::OsString::new();
-      msi_path.push("\"\"\"");
-      msi_path.push(&found_path);
-      msi_path.push("\"\"\"");
-
-      let installer_args = [
-        config.tauri.updater.windows.install_mode.msiexec_args(),
-        config
-          .tauri
-          .updater
-          .windows
-          .installer_args
-          .iter()
-          .map(AsRef::as_ref)
-          .collect::<Vec<_>>()
-          .as_slice(),
-      ]
-      .concat();
-
-      // run the installer and relaunch the application
-      let mut powershell_cmd = Command::new(powershell_path);
-
-      powershell_cmd
-        .args(["-NoProfile", "-WindowStyle", "Hidden"])
-        .args([
-          "Start-Process",
-          "-Wait",
-          "-FilePath",
-          "$Env:SYSTEMROOT\\System32\\msiexec.exe",
-          "-ArgumentList",
-        ])
-        .arg("/i,")
-        .arg(&msi_path)
-        .arg(format!(", {}, /promptrestart;", installer_args.join(", ")))
-        .arg("Start-Process")
-        .arg(current_executable);
-
-      if !current_exe_args.is_empty() {
-        powershell_cmd
-          .arg("-ArgumentList")
-          .arg(current_exe_args.join(", "));
-      }
-
-      let powershell_install_res = powershell_cmd.spawn();
-      if powershell_install_res.is_err() {
-        // fallback to running msiexec directly - relaunch won't be available
-        // we use this here in case powershell fails in an older machine somehow
-        let msiexec_path = system_root.as_ref().map_or_else(
-          |_| "msiexec.exe".to_string(),
-          |p| format!("{p}\\System32\\msiexec.exe"),
-        );
-        let _ = Command::new(msiexec_path)
-          .arg("/i")
-          .arg(msi_path)
-          .args(installer_args)
-          .arg("/promptrestart")
-          .spawn();
-      }
-
-      exit(0);
```

Comment:
this fallback logic however can be replaced with `ShellExecuteW`

Reply:
Thanks for pointing out the problem, it has been fixed